src/service/context/mod.rs

```diff
@@ -151,6 +151,55 @@ impl<S> Context<S> {
     pub fn get<T: Send + Sync + 'static>(&self) -> Option<&T> {
         self.extensions.get::<T>()
     }
+    /// Get a reference to an extension or T's default().
```

Comment:
You miss a line here. It can help to run `just qa` (does require to install just), but you can check the `justfile` to see what it does in case you do not want it. Pretty certain that running `cargo fmt` alone will already add the line here.